webpack.config.js

```diff
@@ -0,0 +1,38 @@
+var path = require('path');
+var webpack = require('webpack');
+var fs = require('fs');
+var copyright = fs.readFileSync('lib/copyright.frag', 'utf8');
+
+module.exports = {
+    entry: './lib/index.js',
+    output: {
+        path: path.join(__dirname, 'dist', 'browser'),
+        filename: 'OktaAuth.min.js',
+        library: 'OktaAuth',
+        libraryTarget: 'umd',
+        umdNamedDefine: true
+    },
+    resolve: {
+      alias: {
+        'ajaxRequest': './reqwestRequest'
+      }
+    },
+    plugins: [
+      new webpack.BannerPlugin(copyright),
+      new webpack.DefinePlugin({
+        SDK_VERSION: JSON.stringify(require('./package.json').version)
+      }),
+      new webpack.optimize.UglifyJsPlugin({
+        compress: {
+          warnings: false
+        },
+        sourceMap: false,
+        comments: function(node, comment) {
+          // Remove other Okta copyrights
```

Comment:
Double checking - can you verify that no other comments got caught here? Looks good, but want to verify.